packages/vite/src/node/server/index.ts

```diff
@@ -603,7 +603,7 @@ async function startServer(
         })
       }
 
-      if (options.open && !isRestart) {
+      if (options.open && options.open !== 'false' && !isRestart) {
         const path = typeof options.open === 'string' ? options.open : base
```

Comment:
If we are going to support `'false'`, I think `'true'` should also be handled here, and default to `base` in that case

Reply:
yep, although `--open true` usually written as `--open`, let me improve it